FakeXrmEasy.Tests.Shared/FakeContextTests/FakeContextTestCreate.cs

```diff
@@ -181,6 +181,26 @@ public void When_creating_a_record_using_dynamic_entities_primary_key_should_be_
         }
 
         [Fact]
+        public void When_creating_a_record_using_early_bound_entities_and_proxytypes_primary_key_should_be_populated()
+        {
+            var context = new XrmFakedContext();
+            context.ProxyTypesAssembly = Assembly.GetAssembly(typeof(Contact));
+            var c = new Contact();
+            c.Id = Guid.NewGuid();
+
+            IOrganizationService service = context.GetFakedOrganizationService();
```

Comment:
Hi Ben,

Just a small comment. There is a general purpose method context.GetOrganizationService() which will eventually replace GetFakedOrganizationService which will be deprecated soon. This was to encourage a unified naming convention for both unit & integration tests (just changes the context class). And so, an integration test (or a test just for troubleshooting a live issue could be easily converted into a unit test by just changing the context class).